src/BlockTypes/MiniCartContents.php

```diff
@@ -95,13 +95,17 @@ protected function enqueue_assets( array $attributes ) {
 						'value'    => $text_color ? $text_color['value'] : false,
 					),
 					array(
-						'property' => 'border-color',
-						'value'    => $text_color ? $text_color['value'] : false,
+						'property' => 'box-shadow',
+						'value'    => $text_color ? "inset 0 0 0 1px {$text_color['value']}" : false,
 					),
 				),
 			),
 			array(
-				'selector'   => '.wc-block-mini-cart__footer .wc-block-mini-cart__footer-actions .wc-block-mini-cart__footer-checkout',
+				'selector'   => array(
+					'.wc-block-mini-cart__footer .wc-block-mini-cart__footer-actions .wc-block-mini-cart__footer-checkout',
+					'.wc-block-mini-cart__footer .wc-block-mini-cart__footer-actions .wc-block-mini-cart__footer-checkout:hover',
+					'.wc-block-mini-cart__footer .wc-block-mini-cart__footer-actions .wc-block-mini-cart__footer-cart.wc-block-components-button:hover',
```

Comment:
Is the extra class (`.wc-block-components-button`) for specificity reasons or it can be removed?

Reply:
I added it to override the core style. But we don't need it this time because we're using internal style which has a higher priority. I removed that class. Thanks so much for the head-up!

Reply:
Nice, thanks! Can it be removed it also from line 91?

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/2036eb6adeafd878a79ba306bdc17d1ff30698bf/src/BlockTypes/MiniCartContents.php#L91